cognee/eval_framework/corpus_builder/corpus_builder_executor.py

```
@@ -54,6 +55,7 @@ async def build_corpus(
 
     async def run_cognee(self, chunk_size=1024, chunker=TextChunker) -> None:
         setup_logging(logging.ERROR)
+        logger = structlog.get_logger()
```

Ruff (0.8.2)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)